src/errors/checkXcmTxInputs.ts

```diff
@@ -16,14 +16,40 @@ export const checkAssetIdInput = (
 	assetIds: string[],
 	relayChainInfo: ChainInfo,
 	specName: string,
-	destChainId: string
+	destChainId: string,
+	xcmDirection: IDirection
 ) => {
 	for (let i = 0; i < assetIds.length; i++) {
 		const assetId = assetIds[i];
 		const parsedAssetIdAsNumber = Number.parseInt(assetId);
-		const notValidNumber = Number.isNaN(parsedAssetIdAsNumber);
+		const invalidNumber = Number.isNaN(parsedAssetIdAsNumber);
+
+		// check that the specific assetId exists for the destChain
+		if (!invalidNumber) {
+			if (xcmDirection.toLowerCase().startsWith('system')) {
+				const systemParachainId = 1000;
```

Comment:
We have a default ID in `consts.ts` called `SYSTEM_PARACHAINS_IDS` with 1 value in it for the id. You may want to use that as it would make everything come from one place.